app/models/credential.rb

```diff
@@ -84,8 +99,7 @@ def panoptes_client_env
 
   def set_expires_at
     unless expires_at.present?
-      payload, _ = JWT.decode token, client.jwt_signing_public_key, algorithm: 'RS512'
-      self.expires_at ||= Time.at(payload.fetch("exp"))
+      self.expires_at ||= Time.at(client.token_expiry)
```

Comment:
I think this can go, we can use the `client.token_expiry` directly against a UTC time instead.

Reply:
correct

Reply:
but we still need to store the `expires_at` value so that we can remove expired tokens in the future